apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -169,23 +174,47 @@ const DepartureTimes = ({
   departuresForDirection,
   onClick,
   alertsForDirection,
+  stopName: stopId,
   overrideDate
 }: DepartureTimesProps): ReactElement<HTMLElement> => {
   const groupedDepartures = groupBy(departuresForDirection, "trip.headsign");
   return (
     <>
-      {Object.entries(groupedDepartures).map(([headsign, departures]) => {
-        return (
-          <div
-            key={`${headsign}-${route.id}`}
-            onClick={() => onClick(route, directionId)}
-            onKeyDown={() => onClick(route, directionId)}
-            role="presentation"
-          >
-            {getRow(headsign, departures, alertsForDirection, overrideDate)}
-          </div>
-        );
-      })}
+      {Object.keys(groupedDepartures).length > 0 ? (
+        <>
+          {Object.entries(groupedDepartures).map(([headsign, departures]) => {
+            return (
+              <div
+                key={`${headsign}-${route.id}`}
+                onClick={() => onClick(route, directionId)}
+                onKeyDown={() => onClick(route, directionId)}
+                role="presentation"
+              >
+                {getRow(headsign, departures, alertsForDirection, overrideDate)}
+              </div>
+            );
+          })}
+        </>
+      ) : (
+        <div
+          key={`${route.direction_destinations[directionId]}-${route.id}`}
+          onClick={() => onClick(route, directionId)}
+          onKeyDown={() => onClick(route, directionId)}
+          role="presentation"
+        >
+          {!isAtDestination(stopId, route, directionId) ? (
+            departureTimeRow(
+              route.direction_destinations[directionId]
+                ? route.direction_destinations[directionId]!
+                : "",
+              [],
+              route.type === 2
+            )
+          ) : (
+            <></>
```

Comment:
Looks like this would create a clickable row that's empty, is that your intention?